include/gDPLoadTextureBlock_Runtime.inc.c

```diff
@@ -0,0 +1,67 @@
+#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_C
+#define GDP_LOADTEXTUREBLOCK_RUNTIME_C
+
+#include "global.h"
+#include "gfx.h"
+
+// The Following arrays must be defined as const in z_lmap_mark.c to appear in rodata
+
+#ifndef GDP_LOADTEXTUREBLOCK_RUNTIME_CONST
+#define ZCONST
+#else
+#define ZCONST const
+#endif
+
+static ZCONST s32 sLoadTextureBlock_siz[] = {
+    G_IM_SIZ_4b,
+    G_IM_SIZ_8b,
+    G_IM_SIZ_16b,
+    G_IM_SIZ_32b,
+};
+static ZCONST s32 sLoadTextureBlock_siz_LOAD_BLOCK[] = {
+    G_IM_SIZ_4b_LOAD_BLOCK,
+    G_IM_SIZ_8b_LOAD_BLOCK,
+    G_IM_SIZ_16b_LOAD_BLOCK,
+    G_IM_SIZ_32b_LOAD_BLOCK,
+};
+static ZCONST u32 sLoadTextureBlock_siz_INCR[] = {
```

Comment:
Can this be `s32` like the others, or perhaps they could all be `u32`?

Reply:
MM doesn't like all `s32`, but is fine with all `u32`.